docs/walkthrough_collaborate.md

```diff
@@ -2,19 +2,72 @@
 title: Collaborate within eScriptorium
 summary: About features related to collaboration in eScriptorium.
 authors:
-    - Alix Chagué
-date: 2023-01-31
+    - Hugo Scheithauer, Alix Chagué
+date: 2023-02-09
 ---
 
-# Walkthrough : collaborate within eScriptorium
+# Walkthrough: collaborate within eScriptorium
 
-!!! warning
-    The section needs to be completed.
+This walkthrough details features related to collaboration in eScriptorium.
```

Comment:
related to collaboration **within** eScriptorium, or to collabor**ating inside** eScriptorium

Reply:
Actually, I wonder it the first sentence should not be closer to something like "eScriptorium allows collaboration between users of the same instance with features such as teams and sharing" (something like that)